docs/configuration-parameters.md

```diff
@@ -147,6 +148,8 @@ This section provides the details of all the configurations required for the Car
 | carbon.query.stage.input.enable | false | Stage input files are data files written by external applications (such as Flink), but have not been loaded into carbon table. Enabling this configuration makes query to include these files, thus makes query on latest data. However, since these files are not indexed, query maybe slower as full scan is required for these files. |
 | carbon.driver.pruning.multi.thread.enable.files.count | 100000 | To prune in multi-thread when total number of segment files for a query increases beyond the configured value. |
 | carbon.load.all.segment.indexes.to.cache | true | Setting this configuration to false, will prune and load only matched segment indexes to cache using segment metadata information such as columnid and it's minmax values, which decreases the usage of driver memory.  |
+| carbon.secondary.index.creation.threads | 1 | Specifies the number of threads to concurrently process segments during secondary index creation. This property helps fine tuning the system when there are a lot of segments in a table. The value range is 1 to 50. |
```

Comment:
looks like this is unused property, couldn't find usage. can we remove it from code also ?

Reply:
its used, please refer org.apache.spark.sql.secondaryindex.rdd.SecondaryIndexCreator#getThreadPoolSize

Reply:
I see, IDE it was showing grey. Indexing problem may be